src/scr_flush_file.c

```diff
@@ -40,6 +45,184 @@
 #error "globals.h accessed from tools"
 #endif
 
+/*
+ * Remove a particular dataset from the flush file.
+ */
+static void
+scr_flush_file_remove_dataset(int id, char* flush_file_path)
+{
+    kvtree* hash = kvtree_new();
+    kvtree_read_file(flush_file_path, hash);
+
+    kvtree_unset_kv_int(hash, SCR_FLUSH_KEY_DATASET, id);
+
+    /* write the hash back to the flush file */
+    kvtree_write_file(flush_file_path, hash);
+
+    /* delete the hash */
+    kvtree_delete(&hash);
+}
```

Comment:
Let's call ``scr_flush_file_dataset_remove`` in place of this.  It can move it to the "flush_nompi" file like the other functions.
https://github.com/LLNL/scr/blob/55c800d60d621d538c48d1db2cc692a07477273a/src/scr_flush_file_mpi.c#L83